src/utils/regexes.rs

```diff
@@ -43,4 +43,16 @@ pub mod request_items {
         QUERY_PARAM_VALUE_REGEX
             .get_or_init(|| Regex::new(r"^(?<key>[ -~]+)==(?<value>[ -~]+)$").unwrap())
     }
+
+    static SINGLE_JSON_VALUE_REGEX: OnceLock<Regex> = OnceLock::new();
+    pub fn single_json_value() -> &'static Regex {
+        SINGLE_JSON_VALUE_REGEX
+            .get_or_init(|| Regex::new(r"^(?<key>[ -~]+):=(?<value>[ -~]+)$").unwrap())
+    }
+
+    static COMBINE_JSON_VALUE_REGEX: OnceLock<Regex> = OnceLock::new();
+    pub fn combine_json_value() -> &'static Regex {
+        COMBINE_JSON_VALUE_REGEX
+            .get_or_init(|| Regex::new(r"^(?<key>[ -~]+):='(?<value>[ -~]*)'$").unwrap())
+    }
```

Comment:
As said in last comment, replace this to a regex to check only the "value" string, and only check if it start and end with a `'` single quote.

Reply:
@talis-fb 

Do you have the good name for this?